src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/Internal/LibuvThread.cs

```diff
@@ -38,9 +38,17 @@ public class LibuvThread : IScheduler
         private ExceptionDispatchInfo _closeError;
         private readonly ILibuvTrace _log;
         private readonly TimeSpan _shutdownTimeout;
+#if DEBUG
+        private readonly string _stackTrace;
```

Comment:
While useful for debugging, I don't like adding this since Environment.StackTrace can really slow tests down. Even though it only affects DEBUG builds, I don't want to slow down development.

This is super trivial to add manually if you are trying to find out which test is crashing. To be useful, you would want to print the stack trace somewhere anyway.